src/components/PackImg.js

```diff
@@ -34,56 +41,20 @@ const PackImg = ({ className, logoUrl, packName }) => {
       css={css`
         color: var(--color-brand-400);
         font-family: var(--code-font);
-        font-size: 7rem;
+        font-size: ${packAcronym.length < 4 ? '6rem' : '4rem'};
+        background-image: url(${DEFAULT_IMAGE});
+        background-repeat: no-repeat;
+        background-position: center;
+        background-origin: content-box;
       `}
     >
-      <div
-        css={css`
-          border: solid var(--color-brand-400) 2px;
-          height: 1px;
-          width: 60px;
-          position: relative;
-          top: 20%;
-          left: 0;
-        `}
-      />
-      <div
-        css={css`
-          border: solid var(--color-green-200) 2px;
-          height: 1px;
-          width: 30px;
-          position: relative;
-          top: 75%;
-          left: 90%;
-        `}
-      />
-      <div
-        css={css`
-          border: solid var(--color-red-300) 2px;
-          height: 1px;
-          width: 20px;
-          position: relative;
-          top: 50px;
-          left: 0px;
-        `}
-      />
-      <div
-        css={css`
-          border: solid var(--color-brand-300) 2px;
-          height: 1px;
-          width: 50px;
-          position: relative;
-          top: 80%;
-          left: 83%;
-        `}
-      />
       <p
         css={css`
           text-align: center;
-          line-height: 150%;
+          line-height: ${packAcronym.length < 4 ? '190%' : '300%'};
```

Comment:
Hopefully this covers the most common acronym lengths. We might be able to calculate a % based on the acronym length. Another possible option is to just wrap to a new line.

This is fine as-is, I'm just thinking out loud.

Reply:
This was removed in favor of using flexbox for centering